static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -142,6 +142,17 @@ class RuleConditionsForm extends PureComponent<Props, State> {
 
   get timeWindowOptions() {
     let options: Record<string, string> = TIME_WINDOW_MAP;
+    const {alertType} = this.props;
+
+    if (alertType === 'custom_metrics') {
+      // Do not show ONE MINUTE interval as an option for custom_metrics alert
+      options = Object.keys(TIME_WINDOW_MAP)
```

Comment:
we often use lodash/omit for this like:
```suggestion
      options = omit(options, TimeWindow.ONE_MINUTE.toString())
```